apps/design/frontend/src/election_info_screen.tsx

```diff
@@ -1,88 +1,98 @@
 import React, { useState } from 'react';
-import {
-  Election,
-  ElectionId,
-  ElectionIdSchema,
-  unsafeParse,
-} from '@votingworks/types';
+import { ElectionIdSchema, unsafeParse } from '@votingworks/types';
 import {
   Button,
   H1,
   MainContent,
   MainHeader,
   SegmentedButton,
 } from '@votingworks/ui';
+import type { ElectionInfo } from '@votingworks/design-backend';
 import { useHistory, useParams } from 'react-router-dom';
 import styled from 'styled-components';
 import { z } from 'zod';
 import { DateWithoutTime } from '@votingworks/basics';
-import { deleteElection, getElection, updateElection } from './api';
+import { deleteElection, getElectionInfo, updateElectionInfo } from './api';
 import { FieldName, Form, FormActionsRow, InputGroup } from './layout';
 import { ElectionNavScreen } from './nav_screen';
 import { routes } from './routes';
 import { ImageInput } from './image_input';
 
-type ElectionInfo = Pick<
-  Election,
-  'title' | 'date' | 'type' | 'state' | 'county' | 'seal'
->;
-
 const SealImageInput = styled(ImageInput)`
   img {
     width: 10rem;
   }
 `;
 
-function hasBlankElectionInfo(election: Election): boolean {
+function hasBlankElectionInfo(electionInfo: ElectionInfo): boolean {
   return (
-    election.title === '' &&
-    election.state === '' &&
-    election.county.name === '' &&
-    election.seal === ''
+    !electionInfo.title &&
+    !electionInfo.state &&
+    !electionInfo.jurisdiction &&
+    !electionInfo.seal
   );
 }
 
 function ElectionInfoForm({
-  electionId,
-  savedElection,
+  savedElectionInfo,
 }: {
-  electionId: ElectionId;
-  savedElection: Election;
+  savedElectionInfo: ElectionInfo;
 }): JSX.Element {
   const [isEditing, setIsEditing] = useState(
     // Default to editing for newly created elections
-    hasBlankElectionInfo(savedElection)
+    hasBlankElectionInfo(savedElectionInfo)
   );
-  const [electionInfo, setElectionInfo] = useState<ElectionInfo>(savedElection);
-  const updateElectionMutation = updateElection.useMutation();
+  const [editedElectionInfo, setEditedElectionInfo] = useState<ElectionInfo>();
```

Comment:
What made you change from using the saved election info as the default state?

Reply:
I think I'd wanted to get rid of `isEditing`, but then found I needed to keep it. I'll try to revert this.

Reply:
[0236383](https://github.com/votingworks/vxsuite/pull/5915/commits/0236383d9f1ee4aa68ce727969824e285a0ede86)